package.json

```diff
@@ -8,7 +8,7 @@
     "@ant-design/icons": "^4.2.2",
     "@craco/craco": "^5.6.4",
     "@okta/okta-react": "^3.0.2",
-    "@okta/okta-signin-widget": "^4.1.2",
+    "@okta/okta-signin-widget": "^5.7.1",
```

Comment:
Why is the version being changed in this PR? Did you merge main branch and perhaps there are some other files/changes that got lost in that merge?